bin/oref0-determine-basal.js

```diff
@@ -105,7 +106,7 @@ if (!module.parent) {
             console.error(msg.msg);
             // console.log(JSON.stringify(msg));
             if (!params['missing-meal-ok']) {
-              errors.push(msg);
+              warnings.push(msg);
```

Comment:
If you'd like a flag for strict mode that's fine, but the default needs to be to warn, not break the loop. This has caused us issues several times due to network comms issues (like wifi captive portals) when configuration is fine.